packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1064,6 +1065,104 @@ ConceptDeclaration
       return result;
     }
 
+BooleanScalar
+   = BooleanType __ d:BooleanDefault? __ {
+      return {
+        $class: "concerto.metamodel@1.1.0.BooleanScalar",
```

Comment:
You can add a property here to indicate that this is a scalar over a `Boolean`, either using the superType, or invent something new, or use a naming convention?

Reply:
Reviewed ✅